1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
 ---
-post_title: Concepts
+post_title: DC/OS Architecture Concepts
 nav_title: Concepts
 menu_order: 5
 ---
-This page contains terms and definitions for DC/OS.
+## <a name="dcos-concepts"></a>DC/OS Concepts
 
-# <a name="adminrouter"></a>Admin Router
+DC/OS is an abstraction around pre-existing components. These terms are sometimes similar to component terms and sometimes they’re new terms meant to abstract organically grown component terms.
 
-The Admin Router runs on the DC/OS master servers to provide a proxy for the admin parts of the cluster.
+### <a name="cluster"></a>Cluster
 
-# <a name="agent"></a> Agent node
+A DC/OS cluster is a set of networked DC/OS nodes with a quorum of master nodes and any number of public and/or private agent nodes.
 
-A Mesos agent node runs a discrete Mesos task on behalf of a framework. It is an agent instance registered with the Mesos master. The synonym of agent node is worker or slave node. See also [private][1] and [public][2] agent nodes.
+### <a name="network"></a>Network
 
-# <a name="bootstrap"></a>Bootstrap node
+DC/OS has two types of networks: infrastructure networks and virtual networks.
 
-The node where a custom DC/OS installation is run. For more information, see the [system requirements](/docs/1.8/administration/installing/custom/system-requirements/#bootstrap-node).
+#### <a name="infrastructure-network"></a>Infrastructure Network
 
-# <a name="cloudtemplate"></a>Cloud template
+An infrastructure network is a physical or virtual network provided by the infrastructure on which DC/OS runs. DC/OS does not manage or control this networking layer, but requires it to exist in order for DC/OS nodes to communicate.
 
-The [cloud templates][3] are optimized to run DC/OS. The templates are JSON-formatted text files that describe the resources and properties.
+#### <a name="virtual-network"></a>Virtual Network
 
-# <a name="mesoscontainerizer"></a>Mesos Containerizer
+A DC/OS virtual network is specifically an overlay network internal to the cluster that connects DC/OS components and containerized tasks running on DC/OS.
 
-The [Mesos Containerizer][4] provides lightweight containerization and resource isolation of executors using Linux-specific functionality such as cgroups and namespaces.
+- The overlay network provided by DC/OS is VXLAN managed by Navstar.
```

Comment:
Yeah, i totally get it. I'm coming from the direction of what I've heard from the PMs - that we should be calling things by feature names in the doc, not using the internal component names. At a certain point customers will need to know both, however using all kinds codenames to discuss concepts may end up confusing people more than helping. IMHO "virtual network service" makes a lot more sense semantically than Navstar.